mlir-tensorrt/compiler/include/mlir-tensorrt-c/Compiler/Compiler.h

```diff
@@ -28,6 +28,8 @@
 #include "mlir-c/Support.h"
 #include "mlir-executor-c/Common/Common.h"
 #include "mlir-executor-c/Support/Status.h"
+#include "mlir-tensorrt-dialect/Utils/Types.h"
+#include "mlir/CAPI/IR.h"
```

Comment:
Both of these inclusions are illegal.  This is the public (i.e. exported) C interface of a shared library.